ofp_netlink.c: do not include netinet/ip.h

The struct iphdr is defined in netinet/ip.h and in linux/ip.h. The
backwards compatible ofp_netlink.c is modified to include linux/ip.h.

http://www.openfastpath.org/pipermail/openfastpath/2016-October/001330.html


--- a/src/ofp_netlink.cold	2017-04-20 16:08:20.942851781 +0800
+++ b/src/ofp_netlink.c	2017-04-20 16:08:39.222851148 +0800
@@ -9,7 +9,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <netinet/in.h>
-#include <netinet/ip.h>
+#include <linux/ip.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 #include <linux/if_arp.h>
